src/main/modules/i18n/index.ts

```diff
@@ -26,6 +26,9 @@ export class I18Next {
           lookupCookie: 'lang',
           caches: ['cookie']
         },
+        interpolation: {
```

Comment:
That could hide the issue of passing user input (with double quotes) through i18n, an issue I fixed in offers the other day.

I wonder whether it is better to leave it on globally and just disable in place where static content has to have double quotes - one place in the app so far.

Reply:
We shouldn't have double escaping in the application, Nunjucks already does it. Having double escaping and disabling it in each place where it is causing problems is unnecessary work.